roles/ci_gen_kustomize_values/tasks/generate_snippets.yml

```diff
@@ -85,13 +66,26 @@
           }}
 
     - name: Generate CI snippet
+      when:
+        - _tmpl_check_path is exists
+      vars:
+        _tmpl_name: >-
+          {{ cifmw_ci_gen_kustomize_values_dt_src_file | basename }}.j2
+        _tmpl_path: >-
```

Comment:
So, some points to take into account:
- the ci_snippet may not need any generation
- for now we only have one VA, and I already said we may want to get a "common/network-values/" tree, since, *apparently* controlplane would be the same for every VA.

So yes, having a `common` tree that will be looked for first, is planed. But since we don't have anything more than one VA, we can't really be sure it will be even used.

Now, the "path" to the template does not rely on the `architecture` tree, but on string we pass - such as the scenario (`hci` for now) and the stage.name (`network-values` for instance).
So anything changing in the architecture repository tree shouldn't affect the way we look for template, unless we decide to change the scenario name (little to no chance I think) or decide to change the stage.name (same - I don't think this would happen, and even so, moving a directory is easy enough).

At some point, decisions has to be made - and we have to enforce some. I'm not happy with the proposed solution, but for now, I don't have anything better.

So, once we have a second, official VA, we will consider the `common` directory and proper lookup updates. But for now... we have hci only on the plate.

Reply:
Added a TODO for next VA iteration. Closing this discussion then,